independent-publisher-connectors/MS Graph Groups and Users/apiDefinition.swagger.json

```diff
@@ -0,0 +1,1151 @@
+            "schema": {
+              "type": "object",
+              "properties": {
+                "@odata.context": {
+                  "type": "string",
+                  "description": "The Odata.context link.",
+                  "title": "Odata Context"
+                },
+                "value": {
+                  "type": "array",
+                  "items": {
+                    "type": "string",
+                    "description": "List of group object ids",
+                    "title": "Group IDs"
+                  },
+                  "description": "value"
+                }
+              }
+            },
+            "headers": {
+              "\"cache-control\"": {
+                "description": "\"cache-control\"",
+                "type": "string"
+              },
+              "\"client-request-id\"": {
+                "description": "\"client-request-id\"",
+                "type": "string"
+              },
+              "\"content-type\"": {
+                "description": "\"content-type\"",
+                "type": "string"
+              },
+              "\"location\"": {
+                "description": "\"location\"",
+                "type": "string"
+              },
+              "\"request-id\"": {
+                "description": "\"request-id\"",
+                "type": "string"
+              }
+            }
+          }
+        },
+        "summary": "Get Member Groups",
+        "description": "Group memberships for a user (member)",
+        "operationId": "GetMemberGroups",
+        "parameters": [
+          {
+            "name": "member-id",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "Object ID of the AAD member(user)",
+            "x-ms-summary": "Object ID of the AAD member(user)",
+            "x-ms-url-encoding": "single"
+          },
+          {
+            "name": "body",
+            "in": "body",
+            "required": true,
+            "schema": {
+              "type": "object",
+              "properties": {
+                "securityEnabledOnly": {
+                  "type": "boolean",
+                  "description": "Do you want to retrieve security enabled groups only?",
+                  "title": "Security Enabled Only (true/false)",
+                  "enum": [
+                    true,
+                    false
+                  ],
+                  "default": false
+                }
+              },
+              "required": [
+                "securityEnabledOnly"
+              ]
+            }
+          }
+        ]
+      }
+    }
+  },
+  "definitions": {},
+  "parameters": {},
+  "responses": {},
+  "securityDefinitions": {
+    "oauth2_auth": {
+      "type": "oauth2",
+      "flow": "accessCode",
+      "authorizationUrl": "https://login.windows.net/common/oauth2/authorize",
+      "tokenUrl": "https://login.windows.net/common/oauth2/authorize",
+      "scopes": {}
```

Comment:
@jagjani03, can you please specify the scopes here? Thank you very much.

Reply:
Hi @Amjed-Ayoub scopes have been updated. Thank you!